RecoMuon/DetLayers/src/MuonGEMDetLayerGeometryBuilder.cc

```diff
@@ -49,6 +65,68 @@ pair<vector<DetLayer*>, vector<DetLayer*> > MuonGEMDetLayerGeometryBuilder::buil
   return res_pair;
 }
 
+MuRingForwardLayer* MuonGEMDetLayerGeometryBuilder::buildLayer0(int endcap,
+                                                                vector<int>& rings,
+                                                                int station,
+                                                                int layer,
+                                                                vector<int>& chambers,
+                                                                vector<int>& rolls,
+                                                                const GEMGeometry& geo) {
+  const std::string metname = "Muon|RecoMuon|RecoMuonDetLayers|MuonGEMDetLayerGeometryBuilder";
+  MuRingForwardLayer* result = nullptr;
+  vector<const ForwardDetRing*> frontRings, backRings;
+
+  LogTrace(metname) << "Starting to Build Layer ";
+  for (std::vector<int>::iterator ring = rings.begin(); ring != rings.end() - 2; ring++) {
```

Comment:
Why "-2"? A comment is deserved, at least
Better to fill the vector with only the needed rings (and not saying "-2" after having filled them)